companies/hoeffner.json

```
@@ -0,0 +1,21 @@
+{
+    "slug": "hoeffner",
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "commerce"
+    ],
+    "name": "Höffner Online GmbH & Co. KG",
+    "address": "Am Rondell 1\n12529 Schönefeld",
+    "phone": "+49 (0)30 37444 -02  und +49 (0)39205 61-333 ",
+    "fax": "+49 30 374448700",
+    "email": " onlineservice@hoeffner.de",
```

```suggestion
    "email": "datenschutz@hoeffner.de",
```
specific email-address for DPO